stylesheets/_component.navigation.scss

```diff
@@ -360,6 +366,115 @@ $nav-color-dark:  darken(color(jadu-blue, dark), 5%) !default;
     }
 }
 
+.nav-tertiary {
+    background-color: $nav-color-dark;
+    box-shadow: inset -4px 0 0 darken($nav-color-dark, 4);
+    left: $nav-primary-width;
+    margin-left: -320px;
+    overflow-x: hidden;
+    overflow-y: auto;
+    position: absolute;
+    top: 0;
+    transition: margin-left 100ms ease-in-out;
+    width: 245px;
+    z-index: $zindex-nav-secondary;
+
+    @include respond-min($screen-desktop) {
+        position: fixed;
+    }
+
+    // cope with presence of scrollbars in IE
+    @include ie-lte(10) {
+        width: 280px;
+    }
+
+    .nav-container {
+        display: none;
+    }
+
+    .nav-container.is-active {
+        display: block;
+        margin: 0 20px;
+    }
+
+    .nav-link {
+        border-top-right-radius: 8px;
+        border-bottom-left-radius: 8px;
+
+        &:hover {
+            background-color: darken($nav-color-dark, 4%);
+            color: white;
+        }
+
+        &:active {
+            background-color: $white;
+            color: $nav-color-dark;
+        }
+
+        span {
+            margin-left: 10px;
+        }
+    }
+}
+
+.nav-quaternary {
+    background-color: $nav-color-darker;
+    box-shadow: inset -4px 0 0 darken($nav-color-dark, 4);
+    left: $nav-primary-width + 3px;
```

Comment:
It is needed for the smooth overlapping animation and style
![overlapping](https://user-images.githubusercontent.com/1425876/28615833-60dd718a-71f2-11e7-8d66-f744fd6b3116.png)